### PR TITLE
Add HTTPS setup description and instructions to the README

### PR DESCRIPTION
**Pull request by ikapelyukhin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,6 +111,30 @@ Connecting an SMT with an RMT this way is not supported.
...
+
+* Certificate: /usr/share/rmt/ssl/rmt-server.crt
+* Private key: /usr/share/rmt/ssl/rmt-server.key
+
+YaST RMT module generates a custom certificate authority which is used to sign HTTPS certificates, which means that in order to register this certificate authority must be trusted by the client machines.
+
+* When registration is performed during installation from the media or with YaST Registration module a message will appear, prompting to trust the server certificate.
+
+* `clientSetup4RMT.sh` script is provided for registering on the command line at the following URL: `http://rmt.hostname/tools/clientSetup4RMT.sh`.
+The script requires only RMT hostname as a mandatory parameter, e.g.:
```
Comment (wstephenson) [inline README.md]: "only the RMT server hostname"

File: README.md [hunk 2]
```diff
@@ -111,6 +111,30 @@ Connecting an SMT with an RMT this way is not supported.
...
+
+    ```bash
+    ./clientSetup4RMT.sh https://rmt.hostname/
+    ```
+
+    Executing this script will import RMT CA's certificate into the trusted store and after that run SUSEConnect to perform the registration.
```
Comment (wstephenson) [inline README.md]: "import the RMT CA's"

"run SUSEConnect to register the client with the RMT"

File: README.md [hunk 3]
```diff
@@ -111,6 +111,30 @@ Connecting an SMT with an RMT this way is not supported.
 
 Available configuration options can be found in the `etc/rmt.conf` file.
 
+The recommended way to perform initial configuration is [YaST RMT module](https://github.com/SUSE/yast2-rmt). The RPM package is [available on OBS](https://build.opensuse.org/package/show/systemsmanagement:SCC:RMT/yast2-rmt).
```
Comment (wstephenson) [inline README.md]: "configuration is using the"

The package is included in sle15 too though, isn't it?

File: README.md [hunk 4]
```diff
@@ -111,6 +111,30 @@ Connecting an SMT with an RMT this way is not supported.
...
+YaST RMT module will take care of configuring SCC credentials, setting up the DB and creating SSL certificates.
```
Comment (wstephenson) [inline README.md]: "The YaST RMT module"

s/DB/database/

File: README.md [hunk 5]
```diff
@@ -111,6 +111,30 @@ Connecting an SMT with an RMT this way is not supported.
...
+
+### SSL certificates & HTTPS
+
+By default access to API endpoints consumed by SUSEConnect is limited to HTTPS only.
+nginx is configured to use SSL certificate and private key from the following locations:
+
+* Certificate: /usr/share/rmt/ssl/rmt-server.crt
+* Private key: /usr/share/rmt/ssl/rmt-server.key
+
+YaST RMT module generates a custom certificate authority which is used to sign HTTPS certificates, which means that in order to register this certificate authority must be trusted by the client machines.
```
Comment (wstephenson) [inline README.md]: "in order to register,"

File: README.md [hunk 6]
```diff
@@ -111,6 +111,30 @@ Connecting an SMT with an RMT this way is not supported.
...
+
+* When registration is performed during installation from the media or with YaST Registration module a message will appear, prompting to trust the server certificate.
```
Comment (wstephenson) [inline README.md]: "or with YaST Registration module,"

File: README.md (hunk 1 above)
Comment (ikapelyukhin) [reply README.md]: Articles are hard when your first language doesn't have any :grimacing:

File: README.md (hunk 3 above)
Comment (ikapelyukhin) [reply README.md]: It is, but we typically have more recent builds in OBS and this is mostly for development/pre-SLES15 testing purposes.

**Pull request merged**